x-pack/plugins/observability_ai_assistant/server/functions/index.ts

```diff
@@ -77,11 +77,15 @@ export const registerFunctions: ChatRegistrationFunction = async ({
         `
             : ''
         }
+
+        The user is able to change the language in which it wants you to reply in the settings page for the Observability AI Assistant, which can be found in the Stack Management app under the option AI Assistants.
```

Comment:
Two comments:

* The docs team has requested we call it the "AI Assistant for Observability" in the UI, so that's what it's labeled like in Stack Management > AI Assistants screen.

* On Serverless there is no Stack Management and no AI Assistants sub option. There, the settings app for the Obs Assistant is under "Project Settings".

Reply:
Fixed in https://github.com/elastic/kibana/pull/176444/commits/0f55e45d7e78043cd7722f6e30ab1e0762151f13